.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,8 +5,9 @@
 
      - If you propose a new feature or improvement, select "master"
        as a target branch;
-     - If this is a bug fix or documentation amendment, select
-       the latest release branch (which looks like "0.xx") -->
+     - If this is a bug fix or documentation amendment, select the latest
+       [release branch](https://github.com/aio-libs/aiohttp/branches) (which
+       looks like "x.x.x") -->
```

Comment:
X.x actualy, like 2.
x.x.x is for release tag like v2.3.2

Reply:
Fixed https://github.com/aio-libs/aiohttp/commit/94656c0a9f93dd2b713141ea72fdb1fe858669af - I expected it will create PR, but commit happened, sorry /: